### PR TITLE
feat: support skip verify tls; support etcd dial timeout(default:10s)

### PR DESCRIPTION
**Pull request by ikaiguang**

### CODE REVIEW

File: conf/config.default.ini
Comment (soyking) [inline conf/config.default.ini:1]: Please revert these unnecessary changes?

Comment (ikaiguang) [reply conf/config.default.ini:1]: Okay, the configuration file has been restored

**Comment by ikaiguang** [comment]
the configuration files have been restored to their default values.
